src/wallet/rpcwallet.cpp

```diff
@@ -1849,9 +1849,77 @@ UniValue backupwallet(const UniValue& params, bool fHelp)
 
     bool bWalletBackupResults = BackupWallet(*pwalletMain, GetBackupFilename("wallet.dat"));
     bool bConfigBackupResults = BackupConfigFile(GetBackupFilename("gridcoinresearch.conf"));
+
+    std::vector<std::string> backup_file_type;
+
+    backup_file_type.push_back("wallet.dat");
+    backup_file_type.push_back("gridcoinresearch.conf");
+
+    std::vector<std::string> files_removed;
+    UniValue u_files_removed(UniValue::VARR);
+
+    bool bManageBackupResults = ManageBackups(GetBackupPath(), backup_file_type, 0, 0, files_removed);
+
+    for (const auto& iter : files_removed)
+    {
+        u_files_removed.push_back(iter);
+    }
+
     UniValue ret(UniValue::VOBJ);
     ret.pushKV("Backup wallet success", bWalletBackupResults);
     ret.pushKV("Backup config success", bConfigBackupResults);
+    ret.pushKV("Manage backup file retention success", bManageBackupResults);
+    ret.pushKV("number_of_files_removed", (int64_t) files_removed.size());
+    ret.pushKV("files_removed", u_files_removed);
```

Comment:
Same key styling suggestion as above 
```suggestion
    ret.pushKV("Files removed", u_files_removed);
```

Reply:
No. If anything we need to be moving towards key names that can be processed easily by machine, including scripts. With scripts it is easier with no spaces, because you are less vulnerable to quoting errors. The issues that are pointed out by #527 remain, and I am not sure really what the right answer is. I am open to dialog on it.

Reply:
@div72, what is your opinion on key names for JSON returns, underlines or spaces? First word capitalized or no? I think we should de-necro #527 and move the dialog there. In the meantime, I will change the underlines back to spaces and also capitalize the first word to conform the keys to the original ones that already exist. However the large discussion needs to be had around this, and we have to consider the breakage factor of changing historical keys...

Reply:
snake_case(all lower, underscore) is better imo. RPC shouldn't need to be human-readable.
> However the large discussion needs to be had around this, and we have to consider the breakage factor of changing historical keys...

You can't make an omelette without breaking some eggs, but I agree that we should discuss it more.

Reply:
I aligned this back to the spaces for now and will close this discussion out. I am going to make a comment in #527 to reactivate this discussion there... personally I like snake case too, and it is consistent with Bitcoin(s) newer standards on member variables, which we are adopting.